lib/_http_outgoing.js

```diff
@@ -78,12 +78,14 @@ exports.OutgoingMessage = OutgoingMessage;
 OutgoingMessage.prototype.setTimeout = function(msecs, callback) {
   if (callback)
     this.on('timeout', callback);
+
   if (!this.socket) {
     this.once('socket', function(socket) {
       socket.setTimeout(msecs);
     });
-  } else
+  } else {
     this.socket.setTimeout(msecs);
+  }
```

Comment:
A little bit confused here, why not just keep it in this pull request? It also goes along with the change [here](https://github.com/iojs/io.js/pull/605/files#diff-286202fdbdd74ede6f5f5334b6176b5cR96), as spurred by [this](https://github.com/iojs/io.js/pull/605#discussion-diff-23589222) conversation. Could I keep it in?